src/gcp/cloudfunctions.ts

```diff
@@ -3,14 +3,16 @@ import * as clc from "cli-color";
 import { FirebaseError } from "../error";
 import { logger } from "../logger";
 import { previews } from "../previews";
-import * as api from "../api";
 import * as backend from "../deploy/functions/backend";
 import * as utils from "../utils";
 import * as proto from "./proto";
 import * as runtimes from "../deploy/functions/runtimes";
 import * as iam from "./iam";
+import { Client } from "../apiv2";
+import api, { functionsOrigin } from "../api";
```

Comment:
If I'm reading this correctly, I don't think we need the `api` bit here:

```js
import { functionsOrigin } from "../api";
```

Reply:
Thanks